protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3143,29 +3143,26 @@ message OrderInfo {
   vega.Side side = 1;
   // Price for the order. The price is an integer, for example `123456` is a correctly
   // formatted price of `1.23456` assuming market configured to 5 decimal places.
-  // This field is an unsigned integer passed as a string and needs to be scaled using the market's decimal places.
   string price = 2;
   // Size remaining.
-  // This field is an unsigned integer passed as a string and needs to be scaled using the market's position decimal places.
   uint64 remaining = 3;
   // Boolean that indicates if it is a market order.
   bool is_market_order = 4;
 }
 
-// Request for the estimate of the margin level and, if available collateral is provided, liqudation price for the specified position.
+// Request for the estimate of the margin level and, if available collateral is provided, liquidation price for the specified position.
 message EstimatePositionRequest {
-  // Market ID.
+  // Market ID in which the position is held.
```

Comment:
```suggestion
  // ID for the market in which the position is held.
```

Reply:
We went through and tried to make everything consistent with XXX ID, and kept ID for/of XXX ... where it sounded more natural. With that in mind, how about `Market ID to estimate the position for`?